lib/OpenQA/Worker/Jobs.pm

```diff
@@ -694,6 +695,20 @@ sub upload_images {
     return !$tx || $tx->success;
 }
 
+sub _strip_path_prefix {
+    my ($path) = @_;
+    my ($vol, $dir, $file) = File::Spec->splitpath($path);
+    my $relpath = $file;
+    my @dirs    = File::Spec->splitdir($dir);
+    for my $d (reverse @dirs) {
```

Comment:
ok, I understand the intention now. This sounds basically impossible to solve :)

You better solve that properly in os-autoinst, the worker will always have to play the guessing game.